src/code/templates/fv-egg-sort-game.js

```diff
@@ -91,8 +91,8 @@ let animationEvents = {
       }},
       hatchDrakeInBasket: { id: 1, complete: false, animate: function() {
         const { scale } = _this.props,
-              leftOffset = -250,
-              topOffset = -40;
+              leftOffset = -255,
+              topOffset = -140;
```

Comment:
As far as I can tell, these should actually already be getting unscaled. Below, we pass in a scale factor, which goes to the `initialElementId` HOC and unscales the values. Something is definitely still off though, you saw the scaling doesn't work as expected